portfolio/src/main/webapp/index.html

```diff
@@ -8,11 +8,18 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
+      <h1>Grace's Website</h1>
+      <p>My name is Grace. I'm a STEP intern. Welcome to my website!</p>
+      <img src="images/propic.JPG" width="300">
+      <p>
+          Learn more about my random <a href="moments.html">moments</a> 
+          in my life, my favorite <a href="quotes.html">quotes</a>,
+          and my important life <a href="lessons.html">lessons</a>.
```

Comment:
Nit: this should only be indented 2 spaces in from the `<p>` tag to match the other indentations in this file.

Reply:
I'm not sure how to change tab to indent 2 spaces rather than 5. I tried searching but it seems this works on other code editors and word rather than cloud shell?

Reply:
What are you using as an editor in cloud shell?